test/common/testinfra.py

```diff
@@ -64,8 +64,8 @@
     'verify/debian-8': [ 'master', 'pulls', ],
     'verify/debian-testing': [ 'master', 'pulls' ],
     'verify/fedora-24': [ ],
-    'verify/fedora-25': [ 'master', 'pulls' ],
-    'verify/fedora-26': [ ],
+    'verify/fedora-25': [ ],
```

Comment:
See above, we probably want to keep testing fedora 25 for a while - especially since we're still pushing updates there.